distributed/http/scheduler/api.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import annotations
+
+import json
+
+from distributed.http.utils import RequestHandler
+
+
+class APIHandler(RequestHandler):
+    def get(self):
+        self.write("API V1")
+        self.set_header("Content-Type", "text/plain")
+
+
+class RetireWorkersHandler(RequestHandler):
+    async def post(self):
```

Comment:
In a true REST API we would not wait for the retirement to finish but the POST would create a background job and return a UUID.
A subsequent GET would allow us to check whether that job completed already. This way the HTTP request does not block. Are we willing to go that far?

Reply:
We weren't aiming for REST here, more of an HTTP RPC. There are a lot of things un-REST-ful about this design. 

But I would be ok with returning early and adding another method to poll for completion. What do you think?